cmd/gardenlet/app/app.go

```diff
@@ -60,6 +59,7 @@ import (
 	gardenletconfigv1alpha1 "github.com/gardener/gardener/pkg/gardenlet/apis/config/v1alpha1"
 	"github.com/gardener/gardener/pkg/gardenlet/bootstrap"
 	"github.com/gardener/gardener/pkg/gardenlet/bootstrap/certificate"
+	bootstrappers2 "github.com/gardener/gardener/pkg/gardenlet/bootstrappers"
```

Comment:
?

Reply:
I don't really know why Goland refactoring decided to rename the package here, will fix it 😅